content/en/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline.md

```diff
@@ -0,0 +1,159 @@
+
+   ![jenkinsfile-online](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/jenkinsfile-online.png)
+
+   {{< notice note >}}
+
+   The field specifies the Jenkinsfile path in the code repository. It indicates the repository’s root directory. If the file location changes, the script path also needs to be changed. 
+
+   {{</ notice >}}
+
+### Step 5: Run a pipeline
+
+1. After a pipeline is created, it displays in the list. Click it to go to its detail page.
+
+2. Click **Run** on the right. In the dialog that appears, select **gitlab-demo** from the drop-down list and add a tag number such as `v0.0.2`. Click **OK** to trigger a new activity.
+
+   ![click-run](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/click-run.png)
+
+   ![select-branch](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/select-branch.png)
+
+   {{< notice note >}}
+
+   The pipeline pauses at the stage `deploy to dev`. You need to click **Proceed** manually. Note that the pipeline will be reviewed three times as `deploy to dev`, `push with tag`, and `deploy to production` are defined in the Jenkinsfile respectively.
+
+   {{</ notice >}}
+
+### Step 6: Check pipeline status
+
+1. In the **Task Status** tab, you can see how a pipeline is running. Check the pipeline running logs by clicking **Show Logs** in the top right corner.
+
+   ![check-log](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/check-log.png)
+
+2. You can see the dynamic log output of the pipeline, including any errors that may stop the pipeline from running. For each stage, you can click it to inspect logs, which can also be downloaded to your local machine for further analysis.
+
+   ![pipeline-logs](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/pipeline-logs.png)
+
+### Step 7: Verify results
+
+1. The Docker image built through the pipeline has been successfully pushed to Docker Hub, as it is defined in the Jenkinsfile. In Docker Hub, you will find the image with the tag `v0.0.2` that is specified before the pipeline runs.
+
+   ![docker-image](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/docker-image.png)
+
+2. At the same time, a new tag has been generated in GitLab.
+
+   ![gitlab-result](/images/docs/devops-user-guide/using-devops/gitlab-multibranch-pipeline/gitlab-result.png)
+
+3. The sample application will be deployed to `kubesphere-sample-dev` and `kubesphere-sample-prod` with corresponding Deployments and Services created.
+
+   | Environment | URL                         | Namespace              | Deployment    | Service       |
+   | ----------- | --------------------------- | ---------------------- | ------------- | ------------- |
+   | Development | `http://{NodeIP}:{$30861}`  | kubesphere-sample-dev  | ks-sample-dev | ks-sample-dev |
```

Comment:
Missing `$` sign before `NodeIP`.

Reply:
Thanks for your comment. I added `$` here and also did the same thing in the doc Create a Pipeline Using a Jenkinsfile.

Reply:
Awesome!